src/lib/dracut/dracut.conf.d/gce.conf

```diff
@@ -0,0 +1,3 @@
+# Include NVMe driver due google_nvme_id script
+# that depends on it
+force_drivers+=" nvme "
```

Comment:
I don't think it matters too much here, but I think the more canonical way to do this is to have it part of `module-setup.sh`:

```sh
installkernel() {
    instmods nvme
}
```

Reply:
yeah. less files would be a `+` of that approach too